packages/react-app/src/InkCanvas.js

```diff
@@ -353,6 +391,18 @@ if (props.mode === "edit") {
           />
         </Col>
     </Row>
+    <Row style={{ width: "90vmin", margin: "0 auto", marginTop:"4vh", justifyContent:'center'}}>
+        <Col span={4}>
```

Comment:
I got a bit of overlap between these buttons, I think you can remove the Col wrapper (it's not doing too much)
![image](https://user-images.githubusercontent.com/9612972/92420048-fd7de300-f168-11ea-9f71-8f0fa74fef4d.png)